ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/utils/StringUtils.java

```diff
@@ -125,7 +125,7 @@ public static String highlightDiagnostic(TextDocument textDocument,
      * @return modified identifier with unicode character
      */
     public static String unescapeUnicodeCodepoints(String identifier) {
-        return IdentifierUtils.unescapeUnicodeCodepoints(identifier);
+        return IdentifierUtils.unescapeBallerina(identifier);
```

Comment:
Why do we call `unescapeBallerina()` inside `unescapeUnicodeCodepoints()`? Those two methods are supposed to two different things.  
In fact, `unescapeBallerina()` calls `unescapeUnicodeCodepoint()` inside.

Reply:
ah. This seems to be a different utils class. Method names are confusing.

Reply:
Yes will change the method names thanx